ext/node/polyfills/_fs/_fs_open.ts

```diff
@@ -125,7 +125,7 @@ export function open(
       if (err) {
         (callback as (err: Error) => void)(err);
       } else {
-        callback(null, res!);
+        callback(null, new res!());
```

Comment:
This is a strange bit of code -- should this be just `res!`, or `new FileHandle(res!)?